ravenframework/JobHandler.py

```diff
@@ -585,8 +620,10 @@ def isFinished(self):
       # that is not done.
       for run in self.__running+self.__clientRunning:
         if run:
-          return False
-
+          if uniqueHandler == run.uniqueHandler:
+            return False
+          elif jobIdentifier is None or run.identifier.startswith(jobIdentifier):
+            return False
```

Comment:
From these lines of code, we will first check the uniqueHandler, if False, then check jobIdentifier, If this is what you mean, could you update the docstring to reflect it?

Reply:
I modified it. in reality only the option of the uniqueHandler is used (so I modified it accordingly)